charts/atlas-cluster/README.md

```diff
@@ -0,0 +1,60 @@
+# Helm Chart: MongoDB Atlas Operator (trial version)
+
+The MongoDB Atlas Operator provides a native integration between the Kubernetes orchestration platform and MongoDB Atlas — the only multi-cloud document database service that gives you the versatility you need to build sophisticated and resilient applications that can adapt to changing customer demands and market trends.
+
+## Quick start
+
+**1.** Register for an Atlas account or log in
+
+**2.** Create API Keys for your organization
+    
+    [Manage Programmatic Access to One Organization.] (https://docs.atlas.mongodb.com/configure-api-access)
+
+**3.** Deploy Atlas Kubernetes Operator
+
+``` 
+helm repo add mongodb https://mongodb.github.io/helm-charts
+helm install atlas-operator --namespace=atlas-operator --create-namespace mongodb/mongodb-atlas-operator
+```
+
+> Note. Operator will watch all available namespaces. In order to watch current namespace only set value `--set watchNamespaces=atlas-operator`
+> operator could only watch all namespaces or just its own namespace. Currently watching other namespaces is not supported
+
+**4.** Deploy MongoDB Atlas Cluster
+
+```
+helm install atlas-cluster \
```

Comment:
when I run this locally I get:
```
Error: failed to download "Project’" (hint: running `helm repo update` may help)
```

got fixed by replacing the single quotes but then:
```
Error: must either provide a name or specify --generate-name
```

Reply:
I see did not provide chart name and chart was not released in the index for some reason. I would need to push new version for this README to work